modules/payment/src/Entity/PaymentMethod.php

```diff
@@ -224,6 +224,29 @@ public function setCreatedTime($timestamp) {
   /**
    * {@inheritdoc}
    */
+  public function getOwnerRemoteId() {
```

Comment:
Is there a point in having these methods? We usually need the remote customer ID before creating a payment method, so we need them on PaymentGatewayBase instead.